x/oracle/keeper/migrations.go

```diff
@@ -18,9 +16,9 @@ func NewMigrator(keeper *Keeper) Migrator {
 
 // Migrate1to2 migrates from version 1 to 2.
 func (m Migrator) Migrate1to2(ctx sdk.Context) error {
-	m.keeper.SetStampPeriod(ctx, 1)
-	m.keeper.SetPrunePeriod(ctx, 1)
-	m.keeper.SetMedianPeriod(ctx, 1)
-	m.keeper.SetHistoricAcceptList(ctx, types.DenomList{})
+	m.keeper.SetHistoricStampPeriod(ctx, 1)
```

Comment:
Does it matter that we are not removing the old values?

Reply:
I'm not actually sure. @robert-zaremba would this be an issue?

Reply:
In migration? But on mainnet, nor canon-2 the medians and historic prices won't be recorded. So there is nothing to remove.
The historic prices should only be recorded (now) when `experimental == true`).